src/main/java/it/gov/pagopa/pu/sil/service/actualization/ActualizationService.java

```diff
@@ -0,0 +1,63 @@
+package it.gov.pagopa.pu.sil.service.actualization;
+
+import it.gov.pagopa.actualization.legacy.dto.generated.Pagamento;
+import it.gov.pagopa.pu.auth.dto.generated.AccessToken;
+import it.gov.pagopa.pu.auth.dto.generated.UserInfo;
+import it.gov.pagopa.pu.organization.dto.generated.OrgSilService;
+import it.gov.pagopa.pu.sil.connector.actualization.LegacyActualizationService;
+import it.gov.pagopa.pu.sil.connector.organization.service.OrgSilServiceComponent;
+import it.gov.pagopa.pu.sil.dto.generated.AmountUpdatesDTO;
+import it.gov.pagopa.pu.sil.service.AccessTokenService;
+import it.gov.pagopa.pu.sil.service.AuthorizationService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+public class ActualizationService {
+  private final OrgSilServiceComponent orgSilServiceComponent;
+  private final LegacyActualizationService legacyActualizationService;
+  private final AccessTokenService accessTokenService;
+
+  public ActualizationService(OrgSilServiceComponent orgSilServiceComponent,
+                              LegacyActualizationService legacyActualizationService,
+                              AccessTokenService accessTokenService) {
+    this.orgSilServiceComponent = orgSilServiceComponent;
+    this.legacyActualizationService = legacyActualizationService;
+    this.accessTokenService = accessTokenService;
+  }
+
+  public AmountUpdatesDTO actualize(Long orgSilServiceId,
+                                    String orgFiscalCode, String nav,
+                                    UserInfo loggedUser, String accessToken) {
+    Long organizationId = AuthorizationService.getOrganizationIdFromOrgFiscalCode(loggedUser, orgFiscalCode);
+    AuthorizationService.validateUserForOrganizationId(organizationId, loggedUser);
+    OrgSilService orgSilService = orgSilServiceComponent.getOrgSilServiceById(orgSilServiceId, accessToken)
+      .orElseThrow(() -> new IllegalArgumentException("Organization service not found"));
+
+    AccessToken loggedUserAccessToken = new AccessToken()
+      .accessToken(accessToken)
+      .tokenType("Bearer");
+
+    AccessToken actualAccessToken = accessTokenService.getAccessToken(orgSilService, loggedUserAccessToken);
```

Comment:
I missed this point... let's pass the String accessToken... not the object
```suggestion
    AccessToken actualAccessToken = accessTokenService.getAccessToken(orgSilService, accessToken);
```